profiles/schema.py

```diff
@@ -0,0 +1,97 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+    vocabulary = graphene.String()
+    code = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        print(dir(self.vocabulary))
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
```

Comment:
pretty sure, that without specifying `name` as a field of this type, graphene will not resolve it, since it is in the `translations` block

Reply:
yeah, the rest endpoint only has [the following fields exposed](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/api.py#L131)